File: exam_2019_july/TravelAgency.cpp
#include <iostream>
#include <string>

using namespace std;

int main()
{
	string town;
	cin >> town;
	
	string extras;
	cin >> extras;
	
	string vip;
	cin >> vip;
	
	int days;
	cin >> days;
	
	if(days < 1){
		
		cout << "Days must be positive number!" << endl;
		return 0;
	}
	
	if(days > 7) days --;

	double price = 0.0;
	
	if( town == "Bansko" || town == "Borovets" ){

			if(extras == "withEquipment"){
	
				price = 100 * days;
				if( vip == "yes" ) price -= price*0.1;
				 
		
			}else if( extras == "noEquipment" ){
	
				price = 80 * days;
				if( vip == "yes") price -= price*0.05;
	
			}else{
				cout << "Invalid input!" << endl;
				return 0;
			}

	}else if(town == "Varna" || town == "Burgas"){

			if(extras == "withBreakfast"){
	
				price = 130 * days;
				if( vip == "yes") price -= price*0.12;
	
			}else if(extras == "noBreakfast"){
	
				price = 100 * days;
				if( vip == "yes") price -= price*0.07;
	
			}else{
	
				cout << "Invalid input!" << endl;
				return 0;
			}

	} else{

			cout << "Invalid input!" << endl;
			return 0;			
	}
	
	cout.setf(ios::fixed);
	cout.precision(2);
		
	cout << "The price is " << price << "lv! Have a nice time!" << endl;
	 
	return 0;
}

File: exam_2019_july/TravelAgency.java
import java.util.Scanner;

public class TravelAgency 
{
    public static void main(String[] args)
    {
        Scanner sc = new Scanner(System.in);

		String town = sc.nextLine();
		String extras = sc.nextLine();
		String vip = sc.nextLine();
		int days = Integer.parseInt(sc.nextLine());
		
		if(days < 1){
			System.out.println("Days must be positive number!");
			return;
		}
		
		if(days > 7) days --;
		
		double price = 0.0;
		
		if( town.equals("Bansko") || town.equals("Borovets") ){

			if(extras.equals("withEquipment")){
	
				price = 100 * days;
				if( vip.equals("yes")) price -= price*0.1;
				 
		
			}else if(extras.equals("noEquipment")){
	
				price = 80 * days;
				if( vip.equals("yes")) price -= price*0.05;
	
			}else{
				System.out.println("Invalid input!");
				return;
			}

		}else if(town.equals("Varna") || town.equals("Burgas")){

			if(extras.equals("withBreakfast")){
	
				price = 130 * days;
				if( vip.equals("yes")) price -= price*0.12;
	
			}else if(extras.equals("noBreakfast")){
	
				price = 100 * days;
				if( vip.equals("yes")) price -= price*0.07;
	
			}else{
	
				System.out.println("Invalid input!");
				return;
			}

		} else{

			System.out.println("Invalid input!");
			return;			
		}
		
		System.out.printf("The price is %.2flv! Have a nice time!", price);
    }
}